features/environment-variables.yml

```diff
@@ -0,0 +1,7 @@
+name: CSS environment variables
```

Comment:
The concern I have with having this separate from the safe-area-inset feature is that you can't use `env()` on its own. It was introduced for safe-area-insets, and nothing else at the time.

Reply:
In talking with @mirisuzanne, `env()` variables come up often in spec discussions, but seldom ends up in the final spec. I merged the `environment-variables` feature with `safe-area-insets`, and think the `environment-variables` group will be useful here. WCO is split into a separate PR.

Reply:
Yeah, I tend to agree with @captainbrosset here that `env()` is only a syntax structure for delivering other features. It's like asking when browsers added support for 'css built-in functions' as a concept, instead of asking about support of each individual function. It makes some sense to teach broadly about how function syntax works, but you would almost never ask when that concept gained support, apart from specific features that use it.